testcharms/charm-hub/charms/juju-qa-test/README.md

```diff
@@ -6,46 +6,46 @@ A container-based V2 metadata charm to use in testing juju.
 
 ## Usage
 
-Basic deploy: <br>
-<code>juju deploy juju-qa-test</code>
+Basic deploy: 
+`juju deploy juju-qa-test`
 
-Set the unit status to the first line of the resource, if available, one time:<br>
-<code>juju config juju-qa-test foo-file=true</code>
+Set the unit status to the first line of the resource, if available, one time:
+`juju config juju-qa-test foo-file=true`
 
-Get your fortune<br>
-<code>juju run-action juju-qa-test/0 fortune</code>
+Get your fortune
+`juju run-action juju-qa-test/0 fortune`
 
 
 ## Version, Channel, Series and History
-|Version  |Channel          |Series				 
-|----      |----            |----               
-| 11|latest/stable|bionic, xenial|
-| 14|latest/candidate|focal, bionic, xenial|
-| 20|latest/edge|groovy, focal, bionic, xenial|
-|8   |2.0/stable  |disco, bionic, xenial, trusty |
-|10      |2.0/edge   |disco, bionic, xenial, trusty   |
+| Version | Channel          | Series                        |
+| ------- | ---------------- | ----------------------------- |
+| 11      | latest/stable    | bionic, xenial                |
+| 14      | latest/candidate | focal, bionic, xenial         |
+| 20      | latest/edge      | groovy, focal, bionic, xenial |
+| 8       | 2.0/stable       | disco, bionic, xenial, trusty |
+| 10      | 2.0/edge         | disco, bionic, xenial, trusty |
```

Comment:
This table needs to be updated, when uploading the charm to charmhub. You can edit the version file.

Reply:
I agree. I was waiting for more instructions here.